### PR TITLE
Add the ability to affix labels to all pull requests created

### PR DESCRIPTION
**Pull request by kaosx5s**
### Description
- What Changed and Why? :
Added the ability to send a list of label names when submitting pull requests.

- Type of change :
  - [x] New feature
  - [ ] Bug fix for existing feature
  - [ ] Code quality improvement
  - [ ] Addition or Improvement of tests
  - [ ] Addition or Improvement of documentation

### CODE REVIEW

File: gordian/gordian.py [hunk 1]
```diff
@@ -84,6 +84,14 @@ def get_basic_parser():
         dest='force_changelog',
         help='Fail if changelog does not exist or cannot be parsed'
     )
+    parser.add_argument(
+        '-l','--labels',
+        required=False,
+        default=None,
```
Comment (jnevelson) [inline gordian/gordian.py:90]: Wha if the default is set to `[]` so that you don't need the check on line 160?

Comment (kaosx5s) [reply gordian/gordian.py:90]: Makes more sense, I'll do that instead.

Comment (kaosx5s) [reply gordian/gordian.py]: Fixed in: https://github.com/argoproj-labs/gordian/pull/22/commits/4a941780056e655665cc8e1ac18ab9b1f10eda67 👍

File: tests/test_gordian.py [hunk 2]
```diff
@@ -18,6 +18,7 @@ def __init__(self, config_file='./tests/fixtures/test_config.yaml', dry_run = Fa
             self.github_api = None
             self.semver_label = None
             self.target_branch = 'master'
+            self.pr_labels = ['test']
```
Comment (jnevelson) [inline tests/test_gordian.py:21]: Can you add a test without this set to ensure that the default `[]` also works?

Comment (kaosx5s) [reply tests/test_gordian.py:21]: Added in: https://github.com/argoproj-labs/gordian/pull/22/commits/2d1f36eb004bc1ba61026cfdd75e3d75bfb395fe 🍰

**Pull request merged**